templates/example_visualization.ipynb

```diff
@@ -6,7 +6,18 @@
    "metadata": {
```

Comment:
<p>Don't forget to erase output before pushing notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>